chapters/resampling.qmd

```diff
@@ -584,6 +586,144 @@ Should we use the bootstrap to compare models? Its small variance is enticing, b
 
 Let's take a look at a few specialized resampling methods. 
 
+
+## Time Series Data {#sec-time-series-resampling}
+
+Time series data [@Hyndmanforecast] are sequential observations ordered over time. The most common example is daily stock prices. These data are a special case due to _autocorrelation_: rows of the data set are not independent. This can occur for many different reasons (e.g. underlying seasonal trends, etc.), but it complicates the resampling process in different ways: 
+
+ - We want to avoid analysis and assessment partitions where older data are in the assessment set (to avoid predicting past data with future data). 
+ - Random sampling might destroy the underlying data structures that can be used to measure specific trends (such as seasonality). 
```

Comment:
```suggestion
 - Placing newer data in the analysis set and older data in the assessment set would not make sense because of the time trends in the data.
```